README.md

```diff
@@ -62,16 +62,21 @@ feat(ui): Add `Button` component.
             fix
             feat
           # Configure which scopes are allowed (newline delimited).
+          # These are regex patterns auto-wrapped in `^ $`.
           scopes: |
             core
             ui
+            JIRA-\d+
+            (?![A-Z])+
```

Comment:
```suggestion
            JIRA-\d+
```

Nice thanks! I meant, should we replace the (?![A-Z])+ example with the JIRA example? Do you see value in having both?

Reply:
I figured keeping both as more explicit examples of regex, to assuage any doubts for what it could be used for, but I can also see how that may be slightly redundant 🤔 If you think it's best to only have one, that's definitely fair!

Reply:
I think I'd only show one example to keep the README slim. I'll commit the suggestion, I hope that's ok!

Reply:
Definitely, make sense!